Formula/ntopng.rb

```diff
@@ -32,6 +32,7 @@ class Ntopng < Formula
   depends_on "automake" => :build
   depends_on "gnutls" => :build
   depends_on "json-glib" => :build
+  depends_on "libsodium" => :build
```

Comment:
```suggestion
  depends_on "libsodium"
```
Declaring this only as a build dependency is giving the following error:
```
==> brew linkage --test ntopng
==> FAILED
Broken dependencies:
  /usr/local/opt/libsodium/lib/libsodium.23.dylib (libsodium)
```

Reply:
applied, thanks!